docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -194,6 +190,15 @@ The response is a JSON object, which contains information associated with the ti
 
 For more details, see the OpenAPI documentation of the API Mediation Layer in the API Catalog.
 
+#### Supported authentication methods
+The API Mediation Layer provides multiple methods which clients can use to authenticate. When the API ML is run as a part of the Zowe all of the following methods are enabled and supported. All the methods are supported at least to some extent with each authentication provider.
```

Comment:
...run as part of Zowe, all of the following....

Reply:
Good catch. Fixed.